projects/plugins/jetpack/extensions/shared/external-media/store/actions.ts

```diff
@@ -0,0 +1,15 @@
+export const SET_AUTHENTICATED = 'SET_AUTHENTICATED';
+
+export type SetAuthenticatedAction = {
+	type: typeof SET_AUTHENTICATED;
+	payload: boolean;
+};
+
+export default {
+	setAuthenticated: ( isAuthenticated: boolean ) => {
```

Comment:
This should also accept a source param, since every source handle the authentication separately.

Otherwise, I think users could use the image block to insert a Pexel image which will call `setAuthenticated( true )` which will cause the `isGooglePhotosConnected` function to return true.